### PR TITLE
Fetch all recipes from database

### PR DESCRIPTION
**Pull request by Michael-Hathaway**
The recipes view will now query the database for all recipes, recipe detail view will query database for specific recipe, and create recipe view now inserts new recipe into the database. you'll need to connect the database/setup your own "dbcon.js" in order to test. I've been using a local copy of the database so far.

### CODE REVIEW

**Comment by Michael-Hathaway** [comment]
I might have also accidentally modified which version of express-handlebars we're using and it might need to be switched back

File: views/layouts/main.handlebars [hunk 1]
```diff
@@ -41,7 +41,7 @@
 		
 	{{!-- include other javascript --}}
 	{{#each js}}
-		<script type="module" src="/javascript/{{this}}"></script>
+		<script src="/javascript/{{this}}"></script>
```
Comment (Vectis99) [inline views/layouts/main.handlebars:44]: Don't quite understand what's going on here but I have never seen the "type" field set to anything other than "text/javascript" or something to that effect in my experience. I didn't know you could omit the field, but if it works it works.

File: routes/indexRouter.js [hunk 2]
```diff
@@ -6,12 +6,25 @@ const router = express.Router();
...
+  res
+    .status(200)
+    .render("login", { css: ["login.css"], js: ["login_form_validation.js"] });
+});
+
+// handle login post request
+router.post("/login", (req, res, next) => {
+  const { username, password } = req.body;
+
+  // TODO: query database to check if user exist
```
Comment (Vectis99) [inline routes/indexRouter.js:25]: Is this what you were working on when you said "I’m gonna start working on the login and registration"? Will that be a separate PR?

File: views/layouts/main.handlebars (hunk 1 above)
Comment (Michael-Hathaway) [reply views/layouts/main.handlebars:44]: I had set the type="module" to try and solve an error when importing something in the client side. I didn't end up being necessary, but i'll add the type="text/javascript" back.

File: routes/indexRouter.js (hunk 2 above)
Comment (Michael-Hathaway) [reply routes/indexRouter.js:25]: yeah! I started to dive into and realized it would be better for me to create a new branch for that feature
